rest-api-spec/src/main/resources/rest-api-spec/test/search/350_matched_queries.yml

```diff
@@ -0,0 +1,103 @@
+setup:
+  - skip:
+      version: " - 2.12.0"
```

Comment:
I think think `skip` section applies to `matched queries with scores` test case, right?

Reply:
yes